app/bundles/EmailBundle/Helper/PlainTextHelper.php

```diff
@@ -208,6 +208,8 @@ class PlainTextHelper
         //  and not constrain text to a fixed-width column.
 
         'base_url' => '',
+
+        'preview_length' => 119, // Maximum length of the preview text
```

Comment:
why do we have the value set to 119 characters?

Reply:
Average character display for gmail webmail client. Based on https://www.emailonacid.com/blog/article/email-marketing/preview-vs-preheader-text-how-long-should-preheader-text-be/